### PR TITLE
Add spell checking for docs.

### PR DESCRIPTION
**Pull request by geeknoid**
This isn't a blocker right now, it just outputs warnings.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -240,14 +240,14 @@ routing_v1alpha3_path := networking/v1alpha3
 routing_v1alpha3_protos := $(shell find networking/v1alpha3 -type f -name '*.proto' | sort)
 routing_v1alpha3_pb_gos := $(routing_v1alpha3_protos:.proto=.pb.go)
 routing_v1alpha3_pb_pythons := $(routing_v1alpha3_protos:.proto=_pb2.py)
-routing_v1alpha3_pb_docs := $(routing_v1alpha3_protos:.proto=.pb.html)
+routing_v1alpha3_pb_doc := $(routing_v1alpha3_path)/istio.routing.v1alpha3.pb.html
```
Comment (rshriram) [inline Makefile]: bad merge? This kind of goes back to single document

File: Makefile [hunk 2]
```diff
@@ -240,14 +240,14 @@ routing_v1alpha3_path := networking/v1alpha3
...
 
-generate-routing-go: $(routing_v1alpha3_pb_gos) $(routing_v1alpha3_pb_docs)
+generate-routing-go: $(routing_v1alpha3_pb_gos) $(routing_v1alpha3_pb_doc)
```
Comment (rshriram) [inline Makefile]: same

File: Makefile [hunk 3]
```diff
@@ -240,14 +240,14 @@ routing_v1alpha3_path := networking/v1alpha3
...
 
-$(routing_v1alpha3_pb_gos) $(routing_v1alpha3_pb_docs): $(routing_v1alpha3_protos)
+$(routing_v1alpha3_pb_gos) $(routing_v1alpha3_pb_doc): $(routing_v1alpha3_protos)
```
Comment (rshriram) [inline Makefile]: and here

File: Makefile [hunk 4]
```diff
@@ -240,14 +240,14 @@ routing_v1alpha3_path := networking/v1alpha3
...
 	## Generate networking/v1alpha3/*.pb.go
 	@$(docker_lock) status
-	@$(docker_gen) $(gogofast_plugin) $(protoc_gen_docs_plugin_for_networking)$(routing_v1alpha3_path) $^
+	@$(docker_gen) $(gogofast_plugin) $(protoc_gen_docs_plugin)$(routing_v1alpha3_path) $^
```
Comment (rshriram) [inline Makefile]: and here

File: Makefile [hunk 5]
```diff
@@ -258,7 +258,7 @@ $(routing_v1alpha3_pb_pythons): $(routing_v1alpha3_protos)
 
 clean-routing:
 	rm -f $(routing_v1alpha3_pb_gos)
-	rm -f $(routing_v1alpha3_pb_docs)
+	rm -f $(routing_v1alpha3_pb_doc)
```
Comment (rshriram) [inline Makefile]: and here

**Comment by geeknoid** [comment]
Sorry, I messed up merge of the Makefile.

**Pull request merged**